examples/with-next-rainbow-button/pages/_app.tsx

```diff
@@ -0,0 +1,34 @@
+import "../styles/global.css";
+import "@rainbow-me/button/styles.css";
+import type { AppProps } from "next/app";
+import { configureChains, createConfig, WagmiConfig } from "wagmi";
+import { mainnet } from "wagmi/chains";
+import { publicProvider } from "wagmi/providers/public";
+import { RainbowButtonProvider, RainbowConnector } from "@rainbow-me/button";
+
+const { chains, publicClient, webSocketPublicClient } = configureChains(
+  [mainnet],
+  [publicProvider()]
+);
+
+const projectId =
+  process.env.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID ?? "YOUR_PROJECT_ID";
+
+const wagmiClient = createConfig({
+  autoConnect: true,
+  connectors: [new RainbowConnector({ chains, projectId })],
```

Comment:
This dev UX isn't too bad; I think it'll work

Reply:
Yeah right. I was looking at handling Context API, but that's way complicated. I think most devs are used to importing connectors in their project so they can pretty much do it easily like that as it is.